### PR TITLE
add qual state enum and db migration

### PR DESCRIPTION
**Pull request by andrewjtimmons**
small pr to add qual state.  I dont have strong opinions on js enum format just did a quick google search.

### CODE REVIEW

File: server/userService.js [hunk 1]
```diff
@@ -3,6 +3,14 @@
 
 var db = require('./db');
 
+// What level of permissions does this user have?
+const QualStateEnum = {
+  banned: 'banned', // not allowed on the platform
```
Comment (dylnclrk) [inline server/userService.js]: 👌

Comment (dylnclrk) [reply server/userService.js]: cool, it's nice to see a friendly string like "banned" when you're trying to debug

Comment (andrewjtimmons) [reply server/userService.js]: <3

**Comment by andrewjtimmons** [comment]
Updated to store the string everywhere so we never have number guessing heck condition

tested on dev.  current users had state qualed.  new user gets state pre qualed.

```
votefwd=# select id, created_at, qual_state from users;
 id |          created_at           |  qual_state   
----+-------------------------------+---------------
  1 | 2018-06-25 21:29:01.506913-07 | qualified
  2 | 2018-06-25 21:30:56.931946-07 | qualified
  5 | 2018-07-19 21:03:05.530616-07 | qualified
  6 | 2018-07-19 21:11:38.492609-07 | qualified
  7 | 2018-07-27 12:00:01.893461-07 | pre_qualified
(5 rows)
```

**Comment by andrewjtimmons** [comment]
@dylnclrk yep! Updated to make it a little nicer

**Pull request merged**